test/amoc_api_helper.erl

```diff
@@ -0,0 +1,54 @@
+-module(amoc_api_helper).
```

Comment:
Fixed in new commit.
Now we disconnect client after each request with `fusco:disconnect/1`.